TCMSApp/src/client/app/runs/runs-exec/runs-edit.controller.js

```diff
@@ -0,0 +1,49 @@
+( function() {
+    'use strict';
+
+    angular
+        .module('app.runsExec')
+        .controller('RunsEditController', RunsEditController);
+
+    RunsEditController.$inject = ['$stateParams','$state', 'logger', 'moment'];
+
+    function RunsEditController($stateParams, $state) {
+        var vm = this;
+        vm.run = JSON.parse(JSON.stringify($stateParams.run));//creating of the deep copy
+        vm.stateName = $state.current.name;
+        vm.addEnvDetail = addEnvDetail;
+        vm.cancel = cancel;
+        vm.save = save;
+        vm.name = undefined;//temporary run name
+        vm.build = undefined;//temporary run build
+        vm.envShort = undefined;//temporary run environment
+        vm.styles = {
+            backColorEdit: '#e6e6e6',
+            backColorRun: '#fff'
```

Comment:
you're setting styles in controller.

much better solution is to set class here. and style the class in css.